docs/adr/0018-implement-special-fields-as-seperate-fields.md

```diff
@@ -0,0 +1,71 @@
+# Implement special fields as seperate fields
+
+* Status: proposed
+* Date: 2021-01-05
+
+## Context and Problem Statement
+
+How to implement special fields in bibtex databases?
+
+## Considered Options
+
+* Special fields as separate fields
+* Special fields as keywords
+* Special fields as values of a special field
+* Special fields as sub-feature of groups
+
+## Decision Outcome
+
+Chosen option: "Special fields as separate fields", because comes out best (see below)
+
+## Pros and Cons of the Options
+
+### Special fields as separate fields
+
+Example:
+```bibtex
+priority = {prio1},
+printed = {true},
+readstatus = {true},
+```
+
+* Good, because groups are another view to fields
+* Good, because a special field leads to a special rendering
+* Good, because groups pull information from the main table
+* Good, because hard-coding presets is easier than generic configuration
+* Good, because direct inclusion in main table
+* Good, because groups are shown with color bars in the main table
+* Good, because there are no “hidden groups” in JabRef
+* Good, because can be easily removed (e.g., by a formatter)
+* Good, because prepares future power of JabRef to make field properties configurable
+* Bad, because bloats BibTeX file (see below)
+* Bad, because requires more timing (see below)
```

Comment:
What do you mean by timing?

Reply:
Typo --> fixed to "Bad, because requires more writing when editing BibTeX manually by hand"